docs/docs/run-node/02-interact-node.md

```diff
@@ -176,36 +184,63 @@ You can replace the query client (here we are using `x/bank`'s) with one generat
 Querying for historical blocks is done by adding the block height metadata in the gRPC request.
 
 ```go
+package main
```

Comment:
same comment as above - made it simpler for noobs to run - maybe there is a better way